source/appendices/e-audit-requirements-and-tests/05-tr-book-tests.rst

```diff
@@ -0,0 +1,145 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2021 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.5 Title Report Audit Tests for Books and Reference_Works
+----------------------------------------------------------
+
+This section of the appendix outlines tests that MUST be run during audits for a Host_Type that is required to offer the Title Report and which includes Book and/or Reference_Work content requiring delivery of the book-related Standard Views of the Title Report (TR_B1, TR_B2, TR_B3).
+
+The Title Report will be COUNTER-compliant for book and/or reference work content if the following audit tests are passed. The book-related Standard Views of the Title Report (TR_B1, TR_B2, TR_B3) will be COUNTER-compliant if the metrics match those in the Title Report, with the appropriate aggregation.
+
+Audit test requirements vary depending on the set up of the platform, as indicated by the Options within the tests below.
+
+Title Report audit tests for books and reference works apply to Host_Types Aggregated_Full_Content, eBook, and eBook_Collection.
+
+Note that Data_Types Book and Reference_Work are included in the audit tests, as Reference_Works are included in the book-related Standard Views of the Title Report.
+
+
+E.5.1 Unique_Title_Investigations and Unique_Title_Requests
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+**Option 1**: Book_Segments and/or Reference_Items are available, users can only access Books and/or Reference_Works segment-by-segment.
+
+* The auditor MUST request 80 Book_Segments and/or Reference_Items, 10 each from 8 different Books and/or Reference_Works.
+* The auditor MUST request 10 Book_Segments and/or Reference_Items, all from the same Book and/or Reference_Work. 
+
+This MUST result in 90 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 9 each of Unique_Title_Investigations and Unique_Title_Requests.
+
+**Option 2**: Only whole Books and/or Reference_Works are available, with no identifiable Book_Segments and/or Reference_Works.
+
+The auditor MUST request 20 Books and/or Reference_Works, twice each. This MUST result in 40 each of Total_Item_Investigations and Total_Item_Requests, and 20 each of Unique_Item_Investigations, Unique_Item_Requests, Unique_Title_Investigations and Unique_Title_Requests.
+
+**Option 3**: Books and/or Reference_Works are available as whole titles, within which Book_Segments and/or Reference_Items can be identified. Alternatively, Books and/or Reference_Works are available as both whole titles and as Book_Segments and/or Reference_Items.
+
+* The auditor MUST request 80 Book_Segments and/or Reference_Items, 10 each from 8 different Books and/or Reference_Works.
```

Comment:
This is only possible if both whole titles and segments are available (second case where this option is applicable).